!SESSION 2015-10-28 10:18:07.068 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

This is a continuation of log file G:\arm\.metadata\.bak_0.log
Created Time: 2015-10-28 11:18:28.122

!ENTRY com.arm.newsfeed 4 0 2015-10-28 11:18:28.122
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-28 12:18:28.238
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-28 13:18:28.156
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more
!SESSION 2015-10-29 10:17:57.412 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY com.arm.newsfeed 4 0 2015-10-29 10:18:06.498
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.python.pydev.core 4 4 2015-10-29 10:18:07.685
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY com.arm.newsfeed 4 0 2015-10-29 11:18:06.035
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-29 12:18:06.303
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-29 13:18:06.182
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-29 14:18:06.385
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-29 15:18:06.186
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-29 16:18:06.266
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more
!SESSION 2015-10-30 10:20:07.574 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY com.arm.newsfeed 4 0 2015-10-30 10:20:15.917
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.python.pydev.core 4 4 2015-10-30 10:20:16.939
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY com.arm.newsfeed 4 0 2015-10-30 11:20:15.472
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-30 12:20:15.783
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-30 13:20:15.605
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-30 14:20:15.928
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-10-30 15:20:15.487
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more
!SESSION 2015-11-10 15:21:35.822 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY com.arm.newsfeed 4 0 2015-11-10 15:21:44.825
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.python.pydev.core 4 4 2015-11-10 15:21:45.762
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY com.arm.newsfeed 4 0 2015-11-10 16:21:44.498
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more
!SESSION 2015-11-11 10:28:10.090 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY com.arm.newsfeed 4 0 2015-11-11 10:28:21.066
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.python.pydev.core 4 4 2015-11-11 10:28:22.103
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY com.arm.newsfeed 4 0 2015-11-11 11:28:20.734
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-11 12:28:20.821
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-11 13:28:20.894
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-11 14:28:20.960
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more
!SESSION 2015-11-27 11:00:04.268 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY com.arm.newsfeed 4 0 2015-11-27 11:00:29.263
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.python.pydev.core 4 4 2015-11-27 11:00:29.327
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY org.eclipse.osgi 2 0 2015-11-27 11:00:33.460
!MESSAGE While loading class "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper", thread "Thread[Thread-8,5,main]" timed out waiting (5000ms) for thread "Thread[Worker-10,5,main]" to finish starting bundle "com.arm.debugger.launcher2_5.15.0.20130620_160734 [321]". To avoid deadlock, thread "Thread[Thread-8,5,main]" is proceeding but "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper" may not be fully initialized.
!STACK 0
org.osgi.framework.BundleException: State change in progress for bundle "reference:file:dropins/plugins/com.arm.debugger.launcher2_5.15.0.20130620_160734.jar" by thread "Worker-10".
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)
Caused by: org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	... 15 more
Root exception:
org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)

!ENTRY com.arm.newsfeed 4 0 2015-11-27 12:00:28.770
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-27 13:00:29.191
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-27 14:00:28.779
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-27 15:00:29.170
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-27 16:00:28.916
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2015-11-27 17:00:30.338
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
java.net.UnknownHostException: forums.arm.com
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:195)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at java.net.Socket.connect(Socket.java:478)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:163)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:388)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:523)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:227)
	at sun.net.www.http.HttpClient.New(HttpClient.java:300)
	at sun.net.www.http.HttpClient.New(HttpClient.java:317)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:970)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:911)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:836)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1172)
	at com.sun.syndication.io.XmlReader.<init>(XmlReader.java:174)
	at com.sun.syndication.io.XmlReader.<init>(XmlReader.java:151)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
!SESSION 2016-01-25 10:10:44.906 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY com.arm.newsfeed 4 0 2016-01-25 10:10:54.717
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.python.pydev.core 4 4 2016-01-25 10:10:55.963
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY org.eclipse.osgi 2 0 2016-01-25 10:10:59.393
!MESSAGE While loading class "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper", thread "Thread[Thread-8,5,main]" timed out waiting (5002ms) for thread "Thread[Worker-1,5,main]" to finish starting bundle "com.arm.debugger.launcher2_5.15.0.20130620_160734 [321]". To avoid deadlock, thread "Thread[Thread-8,5,main]" is proceeding but "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper" may not be fully initialized.
!STACK 0
org.osgi.framework.BundleException: State change in progress for bundle "reference:file:dropins/plugins/com.arm.debugger.launcher2_5.15.0.20130620_160734.jar" by thread "Worker-1".
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)
Caused by: org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	... 15 more
Root exception:
org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)

!ENTRY com.arm.newsfeed 4 0 2016-01-25 11:10:55.072
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2016-01-25 12:10:54.646
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2016-01-25 13:10:54.729
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2016-01-25 14:10:55.039
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2016-01-25 15:10:54.783
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY com.arm.newsfeed 4 0 2016-01-25 16:11:07.212
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more
!SESSION 2016-01-29 13:39:17.638 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY com.arm.newsfeed 4 0 2016-01-29 13:39:28.106
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.python.pydev.core 4 4 2016-01-29 13:39:29.291
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY org.eclipse.osgi 2 0 2016-01-29 13:39:32.796
!MESSAGE While loading class "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper", thread "Thread[Thread-8,5,main]" timed out waiting (5003ms) for thread "Thread[Worker-8,5,main]" to finish starting bundle "com.arm.debugger.launcher2_5.15.0.20130620_160734 [321]". To avoid deadlock, thread "Thread[Thread-8,5,main]" is proceeding but "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper" may not be fully initialized.
!STACK 0
org.osgi.framework.BundleException: State change in progress for bundle "reference:file:dropins/plugins/com.arm.debugger.launcher2_5.15.0.20130620_160734.jar" by thread "Worker-8".
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)
Caused by: org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	... 15 more
Root exception:
org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)
!SESSION 2016-02-01 10:03:40.717 -----------------------------------------------
eclipse.buildId=I20110613-1736
java.version=1.6.0_35
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=ru_RU
Framework arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -product org.eclipse.platform.ide
Command-line arguments:  -pluginCustomization C:\Users\chirik\AppData\Roaming\ARM\DS-5\workbench\pluginCustomization.ini -os win32 -ws win32 -arch x86 -product org.eclipse.platform.ide

!ENTRY org.python.pydev.core 4 4 2016-02-01 10:03:50.709
!MESSAGE sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at com.sun.net.ssl.internal.ssl.Alerts.getSSLException(Alerts.java:174)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.fatal(SSLSocketImpl.java:1764)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:241)
	at com.sun.net.ssl.internal.ssl.Handshaker.fatalSE(Handshaker.java:235)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1206)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.processMessage(ClientHandshaker.java:136)
	at com.sun.net.ssl.internal.ssl.Handshaker.processLoop(Handshaker.java:593)
	at com.sun.net.ssl.internal.ssl.Handshaker.process_record(Handshaker.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:958)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1203)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1230)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1214)
	at sun.net.www.protocol.https.HttpsClient.afterConnect(HttpsClient.java:434)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:166)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1014)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(HttpsURLConnectionImpl.java:230)
	at org.python.pydev.logging.ping.LogPingSender.sendPing(LogPingSender.java:62)
	at org.python.pydev.logging.ping.SynchedLogPing.send(SynchedLogPing.java:164)
	at org.python.pydev.logging.ping.AsyncLogPing.consumeAllCommands(AsyncLogPing.java:131)
	at org.python.pydev.logging.ping.AsyncLogPing.access$0(AsyncLogPing.java:100)
	at org.python.pydev.logging.ping.AsyncLogPing$1.run(AsyncLogPing.java:47)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:289)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:263)
	at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:173)
	at sun.security.validator.Validator.validate(Validator.java:218)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:126)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:209)
	at com.sun.net.ssl.internal.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:249)
	at com.sun.net.ssl.internal.ssl.ClientHandshaker.serverCertificate(ClientHandshaker.java:1185)
	... 17 more
Caused by: java.security.cert.CertPathValidatorException: timestamp check failed
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:139)
	at sun.security.provider.certpath.PKIXCertPathValidator.doValidate(PKIXCertPathValidator.java:330)
	at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:178)
	at java.security.cert.CertPathValidator.validate(CertPathValidator.java:250)
	at sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:275)
	... 24 more
Caused by: java.security.cert.CertificateExpiredException: NotAfter: Thu Sep 26 00:29:33 GMT+03:00 2013
	at sun.security.x509.CertificateValidity.valid(CertificateValidity.java:256)
	at sun.security.x509.X509CertImpl.checkValidity(X509CertImpl.java:568)
	at sun.security.provider.certpath.BasicChecker.verifyTimestamp(BasicChecker.java:157)
	at sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:109)
	at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:117)
	... 28 more

!ENTRY com.arm.newsfeed 4 0 2016-02-01 10:03:50.838
!MESSAGE Error when reading RSS feed "http://forums.arm.com/index.php?/rss/blog/"
!STACK 0
com.sun.syndication.io.ParsingFeedException: Invalid XML: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:182)
	at com.sun.syndication.io.SyndFeedInput.build(SyndFeedInput.java:122)
	at com.arm.newsfeed.rss.AggregatedFeed.fetch(AggregatedFeed.java:226)
	at com.arm.newsfeed.rss.AggregatedFeed.access$3(AggregatedFeed.java:222)
	at com.arm.newsfeed.rss.AggregatedFeed$3.run(AggregatedFeed.java:201)
Caused by: org.jdom.input.JDOMParseException: Error on line 7: The entity name must immediately follow the '&' in the entity reference.
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:468)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:851)
	at com.sun.syndication.io.WireFeedInput.build(WireFeedInput.java:178)
	... 4 more
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:195)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:174)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:388)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1427)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1846)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3033)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:647)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:140)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:808)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:737)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:119)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1205)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
	at org.jdom.input.SAXBuilder.build(SAXBuilder.java:453)
	... 6 more

!ENTRY org.eclipse.osgi 2 0 2016-02-01 10:03:54.758
!MESSAGE While loading class "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper", thread "Thread[Thread-8,5,main]" timed out waiting (5003ms) for thread "Thread[Worker-3,5,main]" to finish starting bundle "com.arm.debugger.launcher2_5.15.0.20130620_160734 [321]". To avoid deadlock, thread "Thread[Thread-8,5,main]" is proceeding but "com.arm.debugger.launcher2.databasewrapper.ConfigurationDatabaseWrapper" may not be fully initialized.
!STACK 0
org.osgi.framework.BundleException: State change in progress for bundle "reference:file:dropins/plugins/com.arm.debugger.launcher2_5.15.0.20130620_160734.jar" by thread "Worker-3".
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)
Caused by: org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	... 15 more
Root exception:
org.eclipse.osgi.framework.internal.core.AbstractBundle$BundleStatusException
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.beginStateChange(AbstractBundle.java:1087)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.start(AbstractBundle.java:297)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:440)
	at org.eclipse.osgi.internal.loader.BundleLoader.setLazyTrigger(BundleLoader.java:268)
	at org.eclipse.core.runtime.internal.adaptor.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:107)
	at org.eclipse.osgi.baseadaptor.loader.ClasspathManager.findLocalClass(ClasspathManager.java:462)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.findLocalClass(DefaultClassLoader.java:216)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:400)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:429)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.baseadaptor.DefaultClassLoader.loadClass(DefaultClassLoader.java:107)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at com.arm.debugger.launcher2.LauncherPlugin$1.run(LauncherPlugin.java:154)
	at java.lang.Thread.run(Thread.java:662)
